packages/table/src/regions.ts

```diff
@@ -164,6 +165,13 @@ export class Regions {
     }
 
     /**
+     * Returns a region containing the entire table.
+     */
+    public static table(): IRegion  {
+        return {};
```

Comment:
Seems like this might make some headaches for us in the future. Can you please update the docs examples about regions?